pkgs/by-name/fr/framework-tool/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, rustPlatform, fetchFromGitHub, pkg-config, udev }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "framework-tool";
+
+  # Latest stable version 0.1.0 has an ssh:// git URL in Cargo.lock,
+  # so use unstable for now
+  version = "unstable-2023-11-14";
+
+  src = fetchFromGitHub {
+    owner = "FrameworkComputer";
+    repo = "framework-system";
+    rev = "491a587342a5d79366a25d803b7065169314279c";
+    hash = "sha256-qDtW4DvY19enCfkOBRaako9ngAkmSreoNWlL4QE2FAk=";
+  };
+
+  cargoLock = {
+    lockFile = "${src}/Cargo.lock";
```

Comment:
This causes IFD (import from derivation), which is not allowed in nixpkgs, you may copy the lockfile instead.

Reply:
Fixed, thanks!